### DISCUSSION TITLE
How to expose exemplar metrics with auto-instrument

### DISCUSSION THREAD

**Discussion by mvkrishna86** [open-telemetry/opentelemetry-java-instrumentation#4655] [Q&A]
Hi,
May I know how can I auto-instrument the java code for exposing metrics along with exemplars?
I tried the procedure mentioned here: https://github.com/prometheus/client_java/blob/master/OTEL_EXEMPLARS.md. But in the code, it looks like we have to create the metrics and add the instrumentation before every call. 

It is mentioned here: https://github.com/prometheus/client_java/blob/master/OTEL_EXEMPLARS.md, that we can add the exemplars without changing the code. But May I know how to do that?

**Comment by irizzant** [comment]
I'd definitively like to have guidance on this subject too. I also opened the following https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/4616

**Comment by jkwatson** [comment]
I think this is a discussion for the instrumentation repository, rather than the SDK, correct? Assuming you want to configure the javaagent for this, rather than manually configuring the SDK, that is.

**Reply by irizzant** [reply]
we need to configure the Java auto instrumentation via Java agent to expose Exemplars via Prometheus exporter, but we lack documentation for this

**Reply by jkwatson** [reply]
right. So, the issue you created over in the instrumentation repo is the right place for this to be discussed, rather than here.

**Reply by irizzant** [reply]
right, but I found this discussion already here with the very same subject and I just replied to it so I guess it should have been moved before eventually

**Reply by jkwatson** [reply]
I'm going to move this over to the instrumentation repo. :)
